test/general-tests/constructor-test/no_constructor.cairo

```diff
@@ -0,0 +1,12 @@
+#[contract]
+mod Contract {
+
+    struct Storage {
+        balance: felt252,
+    }
+
+    #[external]
+    fn increase_balance(amount1: felt252, amount2: felt252) {
```

Comment:
Did it not allow an empty constructor? If so, you can add a comment line documenting the purpose of this file and why it cannot be empty.

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/377/commits/81d8dd0f8431b12eac70fcdd226edc426bae3432